mysql-test/include/filter_file.inc

```diff
@@ -114,7 +123,7 @@ perl;
   {
     chomp;
     s/\015?\012?$//;
-    if (!%column_names)
+    if (!%column_names && $skip_column_names == 0)
```

Comment:
This all looks reasonable to me but I'm not terribly familiar with this functionality in mtr internals. @laurynas-biveinis do you have a compelling opinion?